.idea/codeStyles/Project.xml

```diff
@@ -1,6 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <JSCodeStyleSettings version="0">
```

Comment:
What version of IDEA you are running? I've got 2018.3.5

Reply:
I have 2018.2.4

Reply:
Please upgrade!